src/webgpu/api/validation/gpu_external_texture_expiration.spec.ts

```diff
@@ -322,42 +322,3 @@ g.test('import_from_different_video_frame')
       t.submitCommandBuffer(bindGroup, true);
     });
   });
-
-g.test('device_destroyed_before_importing_video_frame')
-  .desc(
-    `
-    Tests that imported GPUExternalTexture from destroyed device and submit
-    it should result in an error.
```

Comment:
Agree this description looks wrong, but the test itself doesn't seem to actually be checking for an error. I think it's OK to test this behavior. Even though we can't assert anything, we can still detect if the browser crashes or something.

Reply:
Ok, I think I'd like to move this test case to a proper place here (https://github.com/gpuweb/cts/blob/904c74af6fee0530ce33434974aa0df6316fdef1/src/webgpu/api/validation/state/device_lost/destroy.spec.ts#L44)

Reply:
That makes sense.